native/packages/react-native-bpk-component-button/readme.md

```diff
@@ -41,42 +41,42 @@ export default class App extends Component {
     return (
       <View style={styles.container}>
         <BpkButton
-          type="primary"
+          type={BUTTON_TYPES.primary}
```

Comment:
I'm a bit sceptical about this, what is the advantage of this over strings in javascript?

Reply:
I did this to be consistent with other components, e.g. badges: https://github.com/Skyscanner/backpack/blob/master/native/packages/react-native-bpk-component-badge/readme.md

Bear in mind that this was the first non-trivial RN component we made, so it's a little behind in terms of best practices and coding style etc.

Reply:
If that is the standard then 🤷‍♂️ . I just don't see any reason for doing it in javascript because you end up typing more and there's no static validation to ensure you added to right value, i.e. you can just add `BUTTON_TYPES.pimary` and it will only fail during runtime. 

If we use flow to ensure `type` only accepts some specific set of strings we actually static validation.